### PR TITLE
Cleanups, fixes, Makefile, …

### PR DESCRIPTION
**Pull request by liskin**
#### [Clean up Gemfile and _config.yml](../commit/62bbc7d44cb43b71e2af24a8dc5340227626f5c4)

Drop plugins that are included in github-pages by default, drop stuff
that isn't needed/used (minima, Windows-specific gems), drop baseurl.

The thinning of Gemfile may actually be necessary, as bundler's
dependency resolution sometimes installs a horribly old version of
github-pages when it starts the resolution with jekyll-default-layout,
as github-pages depends on an older version of jekyll-default-layout,
and its resolution algorithm then goes all crazy.

#### [Speed up Jekyll build](../commit/38fb561f2ac5da700745054a6e9af06235bea25f)

We're not using any Jekyll themes and this cuts the build time from 1.5
seconds to a little over 100 milliseconds.

#### [Fix broken links to non-existent intro.html](../commit/056c9bba11ed2040767091ac719b2c468f974b9f)

What used to be intro.html is now install-instructions.md.

#### [Workaround for livereload @import bug](../commit/d50b38e4c9f8bd580aa9e87199d0a98722a68ed0)

Gets rid of the annoying

    [2021-06-04 10:27:26] ERROR `/font-raleway.css' not found.

because livereload incorrectly handles relative @import in CSS.

Related: https://github.com/lepture/python-livereload/issues/248

#### [Relative links to images](../commit/0888cc30a8fa6accfd69d25ad9322b18d8efb0ea)

site.url is hardcoded to https://xmonad.org, even when serving locally
or from a fork at GitHub.

#### [Add GitHub Actions workflow to check links (both internal and external)](../commit/364c571661da937ba0133d12833bab40bebe2fc6)

"Stolen" from https://github.com/liskin/work.lisk.in/blob/c6ad22896ad933aed93cc752e0373d4bdb3bc3a8/.github/workflows/linkchecker.yaml

#### [Fix broken links to dzen and rxvt-unicode](../commit/f0d13ffe2b37b8cb8eb6183a855f0d62ef3efa8d)

### CODE REVIEW

**Comment by liskin** [comment]
GitHub says the branch hasn't been deployed but it has, here: https://liskin.github.io/xmonad-web/tour.html

File: download.md [hunk 1]
```diff
@@ -71,6 +71,6 @@ cabal install https://github.com/xmonad/xmonad-contrib/archive/master.tar.gz
 Here are some tools we've found work well with xmonad:
 
 *   [dmenu](https://tools.suckless.org/dmenu/), a program launcher
-*   [dzen](https://gotmor.googlepages.com/dzen), an extensible status bar
+*   [dzen](https://robm.github.io/dzen/), an extensible status bar
 *   [xmobar](https://hackage.haskell.org/package/xmobar), an extensible status bar
```
Comment (slotThe) [inline download.md]: I wonder of hackage is the right place to link to concerning xmobar.  This should probably go to either GitHub or `xmobar.org`

**Review by slotThe — APPROVED** [review]
I like this!

Can confirm steps in the README work for me when building with the new Makefile

**Review by TheMC47 — APPROVED** [review]
Thank you for this much-needed cleanup! At least some of us have a clue on what they're doing :grinning:

File: download.md (hunk 1 above)
Comment (liskin) [reply download.md]: xmobar.org seems to be from before https://github.com/jaor/xmobar/commit/fa8c748b5b2e3a1a6f6add0ad6bc2a8e56ddcc28, which seems to also be the point when Hackage stopped rendering xmobar's readme, so yeah, linking to GitHub is now the best option.

**Pull request merged**